package.json

```diff
@@ -10,13 +10,19 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "node_modules/.bin/sequelize db:migrate",
```

Comment:
You do not need to include `node_modules/.bin/` - npm takes care of figuring out what thing we want to execute...